lib/tasks/heroku_platform_api.rake

```diff
@@ -0,0 +1,28 @@
+namespace :heroku_platform_api do
+  desc 'Restart heroku web dyno'
+
+  # To schedule automatic restarts at 5AM Eastern Time and 5PM Eastern Time,
+  # set two tasks to run hourly:
+  #
+  # $ rake 'heroku_platform_api:restart_web_dynos[5]'
+  # $ rake 'heroku_platform_api:restart_web_dynos[17]'
+  #
+  # Good blog post on rake tasks with arguments from Thoughtbot:
+  # https://robots.thoughtbot.com/how-to-use-arguments-in-a-rake-task
+
+  task :restart_web_dynos, [:hour] => :environment do |t, args|
+    target_hour = args[:hour].to_i
+    now_hour = Time.now.in_time_zone('Eastern Time (US & Canada)').hour
```

Comment:
Would it simplify to run the task every hour, and write `if now_hour == 5 || now_hour == 17` in the code?

Reply:
Yeah, hardcoding in the now_hour values would work too. My thinking here is that allowing `hour` to be set as an argument to the rake task in the Heroku Scheduler would make it simple to adjust the timing of the restarts.

 If we want to change how many times per day we restart, or what times we restart, we just log into the Heroku UI and change the argument on the task. No PR, merge, deploy process needed.